jwst/pipeline/level2association.py

```diff
@@ -0,0 +1,100 @@
+"""Open a Level2 association given various inputs"""
```

Comment:
So, I did everything: Genericized the whole thing and put it back into associations. Will be evident in the push that is about to happen...once the @#$@#$@ tests run.